server/database/index.js

```diff
@@ -34,10 +34,10 @@ const updateProduct = (request, response) => {
     pool.query('UPDATE products SET shop_quantity = $1 WHERE id = $2', [updatedShopQuantity, id],
         (error, results) => {
             if (error) {
-                return "Error"
+                return response.status(404).send(`Error updating product with id ${id}`)
             }
-            response.status(200).send(`Product with ID: ${id} modified with shop_quantity of ${updatedShopQuantity}`)
+            return response.status(200).send(`Product with ID: ${id} modified with shop_quantity of ${updatedShopQuantity}`)
```

Comment:
In the future, if you have many strings being interpolated and used in different places of your code, you can think of having a separate module that would be responsible for building strings. This way, if you repeat the same message in different places and then you decide to change the message, you need to change it only in one place :D Now it would probably be an overkill, but it's something to think of in the future :)

Reply:
This is a great tip! Thank you, Justyna :)